website/versioned_docs/version-1.3.0/launcher.md

```diff
@@ -0,0 +1,116 @@
+---
+id: version-1.3.0-launcher-reference
+title: Launcher Reference
+sidebar_label: Built-in Launcher
+original_id: launcher-reference
+---
+
+The launcher is a lightweight artifact (~122K) that installs and configures
+bloop. The goal of the launcher is to relieve clients of the burden of
+installing and managing the lifetime of background build servers.
+
+The launcher implements the [Build Server Discovery protocol][server-discovery]
+which allows clients to establish a bsp connection and communicate with the
+server via stdin and stdout.
+
+## `launcher`
+
+#### Usage
+
+**Synopsis**: `launcher [JVM_OPTS...] [FLAGS...] BLOOP_VERSION`
+
+* `BLOOP_VERSION` must be a bloop version >= 1.2.0. This version is only
+  used in case an installation is tried.
+
+  > There is no guarantee that the server provided to the client will be of this version.
+
+* `JVM_OPTS` must be valid JVM arguments prefixed with `-J`.
+
+#### Flags
+
+<dl>
+<dt><code>--skip-bsp-connection</code> (type: <code>bool</code>)</dt>
+<dd><p>Do not attempt a bsp connection, but exit as soon as bloop has been installed and a server is running in the background.</p></dd>
+</dl>
+
+## Description
+
+The high-level logic steps of the launcher can be summarized as follows:
+
+1. If bloop is not detected in the `$PATH` or `$HOME/.bloop/`, install it.
+   * If the universal installation (that requires `curl` and `python`) fails,
+     then we attempt to resolve bloop and launch an embedded server with
+     a concrete bsp command. This operation is only atempted iff
```

Comment:
iff -> if

Reply:
This is intended, it's a shortcut for if and only if, but I've expanded it